packages/@uppy/aws-s3-multipart/src/createSignedURL.js

```diff
@@ -0,0 +1,98 @@
+function createCanonicalRequest ({
+  method = 'PUT',
+  CanonicalUri = '/',
+  CanonicalQueryString = '',
+  SignedHeaders,
+  HashedPayload,
+}) {
+  const headerKeys = Object.keys(SignedHeaders).map(k => k.toLowerCase()).sort()
```

Comment:
does `sort` work as expected here? It's generally not good without passing an explicit sorting function

Reply:
It's good, it can only sort strings but that's what we want from it.